setup.py

```diff
@@ -7,75 +7,6 @@
 a shorter time. It eventually makes your day-to-day data analytics work more
 efficient and productive.
 """
```

Comment:
Since the config file no longer refers `__doc__`, we can delete the top-level statements.